app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryUiRenderer.kt

```diff
@@ -14,6 +14,7 @@ class CustomDrugEntryUiRenderer(
     } else {
       ui.hideProgressBar()
       ui.showCustomDrugEntryUi()
+      ui.showKeyboard()
```

Comment:
Ideally, this should have been a UI Action/View effect. Now show keyboard method gets invoked every time there is a model change.

You can update this in a different PR